docs/cip/README.md

```diff
@@ -937,12 +863,16 @@ proof that the EB has received sufficient committee approval.
 
 <a id="certificate-inclusion" href="#certificate-inclusion"></a>**Certificate
 Inclusion**: Block producers creating new RBs include certificates for EBs where
-the full stage duration ($L_\text{vote}$ slots) has elapsed since the EB's
-creation (step 12). The producer is permitted to also announce a new EB
-extending their RB. Including a certificate resets the $L_\text{recover}$
-rolling window, either extending an existing enhanced throughput period or
-starting a new one (as detailed in
-[Rolling Window Rules](#rolling-window-rules)).
+at least $L_\text{vote} + L_\text{diff}$ slots have elapsed since the slot of
+the RB that announced the EB (step 12). This timing constraint ensures the
+certified EB has had sufficient time to diffuse throughout the network.
+
+**Transaction Selection**: When producing an RB that includes an EB certificate,
+the block contains only the certificate. When producing an RB without a
+certificate, the producer includes transactions from their mempool only if they
+can validate all transactions against the complete ledger state (including all
+certified EBs). If missing any certified EB data, the producer creates an empty
+RB.
```

Comment:
This paragraph is confusing. It switches from passive to active and the conditions are not clear enough to convince it covers all cases.